airbyte-integrations/connectors/source-asana/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,84 @@
+{"stream": "projects", "data": {"gid": "1201856001387054", "archived": false, "color": "dark-green", "created_at": "2022-02-22T23:20:52.238Z", "current_status": null, "custom_fields": [], "custom_field_settings": [{"gid": "1201856001387132"}, {"gid": "1201856001387112"}, {"gid": "1201856001387116"}, {"gid": "1201856001387139"}, {"gid": "1201856001387164"}], "default_view": "list", "due_on": null, "due_date": null, "followers": [{"gid": "1201617089911900"}, {"gid": "1200344886012993"}, {"gid": "1201607360212792"}], "html_notes": "<body>Standardize your RFP and evaluation process with our template to ensure you\u2019re fully prepared to receive and evaluate proposals\u2014all in the same place\u2014so you can pick the best organization for the job.</body>", "icon": "page_layout", "members": [{"gid": "1201617089911900"}, {"gid": "1200344886012993"}, {"gid": "1201607360212792"}], "modified_at": "2022-05-01T07:33:20.428Z", "name": "Speaking Engagements ", "notes": "Standardize your RFP and evaluation process with our template to ensure you\u2019re fully prepared to receive and evaluate proposals\u2014all in the same place\u2014so you can pick the best organization for the job.", "owner": {"gid": "1201617089911900"}, "permalink_url": "https://app.asana.com/0/1201856001387054/1201856001387054", "public": true, "resource_type": "project", "start_on": null, "team": {"gid": "1200344940838474"}, "workspace": {"gid": "1199152354181012"}}, "emitted_at": 1668517065006}
```

Comment:
I'm afraid this file potentially exposes sensitive internal data as the account used for testing is not a sandbox one. Please delete it.

Reply:
@alafanechere Should we move this certification to `on hold` until the dedicated sandbox account is up and running? 
or should we skip checking `expected records` with bypass reason? there is also a problem with `two sequential reads` since we're testing the active valid account and data is not frozen.

Reply:
I prefer to keep this certification `on hold`, as a dedicated test account will provide better testing guarantees. But I will let @YowanR  decide, I'm not sure of the efforts required for properly seeding a sandbox account.